apps/etterlatte-beregning/src/main/kotlin/beregning/grunnlag/BeregningsGrunnlagRoutes.kt

```diff
@@ -28,71 +25,53 @@ fun Route.beregningsGrunnlag(
             withBehandlingId(behandlingKlient, skrivetilgang = true) { behandlingId ->
                 val forrigeBehandlingId = call.uuid("forrigeBehandlingId")
 
+                logger.info("Dupliserer beregningsgrunnlag for behandling $behandlingId fra $forrigeBehandlingId")
                 beregningsGrunnlagService.dupliserBeregningsGrunnlag(behandlingId, forrigeBehandlingId, brukerTokenInfo)
 
                 call.respond(HttpStatusCode.NoContent)
             }
         }
 
-        post("/{$BEHANDLINGID_CALL_PARAMETER}/barnepensjon") {
+        post("/{$BEHANDLINGID_CALL_PARAMETER}") {
             withBehandlingId(behandlingKlient, skrivetilgang = true) { behandlingId ->
+                logger.info("Lagrer beregningsgrunnlag for behandling $behandlingId")
                 val body = call.receive<LagreBeregningsGrunnlag>()
 
-                when {
-                    beregningsGrunnlagService.lagreBeregningsGrunnlag(
-                        behandlingId,
-                        body,
-                        brukerTokenInfo,
-                    ) -> call.respond(HttpStatusCode.NoContent)
-
-                    else -> call.respond(HttpStatusCode.Conflict)
+                val lagretBeregningsGrunnlag =
+                    beregningsGrunnlagService
+                        .lagreBeregningsGrunnlag(
+                            behandlingId,
+                            body,
+                            brukerTokenInfo,
+                        )
+
+                if (lagretBeregningsGrunnlag != null) {
```

Comment:
Pirk, men kan til fordel bytte ut if med `let`
```
beregningsGrunnlagService
                        .lagreBeregningsGrunnlag(
                            behandlingId,
                            body,
                            brukerTokenInfo,
                        )?.let ......
```

Reply:
Liker ikke `let`, det blir for kryptisk imo.
Enten standard nullsjekk eller noe ala `call.respond(HttpStatusCode.OK, grunnlag ?: HttpStatusCode.NoContent)`